functions/slack/search.go

```diff
@@ -46,14 +48,21 @@ type Message struct {
 // by a Slack command.
 func KGSearch(w http.ResponseWriter, r *http.Request) {
 	setup(r.Context())
+
+	bodyBytes, _ := ioutil.ReadAll(r.Body)
```

Comment:
Check error. Also, just `body`?

Reply:
Just body, r.ParseForm depletes it but VerifyWebhookSigning needs it as commented on line 63